web/src/components/FireTable.tsx

```diff
@@ -1,32 +1,31 @@
-import { Paper, Table, TableContainer, ThemeProvider, Theme, StyledEngineProvider } from '@mui/material'
-import makeStyles from '@mui/styles/makeStyles'
+import { Paper, Table, TableContainer, ThemeProvider, StyledEngineProvider } from '@mui/material'
+import { styled } from '@mui/material/styles'
 import { fireTableTheme } from 'app/theme'
 import React from 'react'
 
-declare module '@mui/styles/defaultTheme' {
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  interface DefaultTheme extends Theme {}
-}
+const PREFIX = 'FireTable'
 
-interface FireTableProps {
-  ariaLabel: string
-  children?: React.ReactNode
-  testId?: string
+const classes = {
+  tableContainer: `${PREFIX}-tableContainer`
 }
 
-const useStyles = makeStyles(() => ({
-  tableContainer: {
+const StyledPaper = styled(Paper)(() => ({
```

Comment:
Sorry, another random question...are there criteria that define when a styled component should be instantiated outside the scope of the functional component versus inside the as is done in `Button.tsx`?

Reply:
No idea! I ran the code mod and if it worked I kept it, otherwise I manually redid it. I've noticed the components that have key, value object classes don't work often 🤷